### ISSUE TITLE
Updating v5.28 → v5.30 fails: errno -13/Error: EACCES to wekan-5.30/bundle/programs/server/package.json

### ISSUE THREAD

**Issue by BabyFnord** [BabyFnord/update_wekan#4]
When trying to update wekan v5.28 → v5.30 and after inflating wekan's main.js, `wekan-5.30/bundle/programs/server/package.json` can't be accessed:

> inflating: /home/USER/wekan/wekan-5.30/bundle/main.js
> npm WARN deprecated har-validator@5.1.5: this library is no longer supported
> npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
> npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
> npm WARN deprecated node-pre-gyp@0.14.0: Please upgrade to @mapbox/node-pre-gyp: the non-scoped node-pre-gyp package is deprecated and only the @mapbox scoped package will recieve updates in the future
> npm ERR! code EACCES
> npm ERR! syscall open
> npm ERR! path /home/USER/wekan/wekan-5.30/bundle/programs/server/package.json
> npm ERR! errno -13
> npm ERR! Error: EACCES: permission denied, open '/home/USER/wekan/wekan-5.30/bundle/programs/server/package.json'
> npm ERR!  [Error: EACCES: permission denied, open '/home/USER/wekan/wekan-5.30/bundle/programs/server/package.json'] {
> npm ERR!   errno: -13,
> npm ERR!   code: 'EACCES',
> npm ERR!   syscall: 'open',
> npm ERR!   path: '/home/USER/wekan/wekan-5.30/bundle/programs/server/package.json'
> npm ERR! }
> npm ERR!
> npm ERR! The operation was rejected by your operating system.
> npm ERR! It is likely you do not have the permissions to access this file as the current user
> npm ERR!
> npm ERR! If you believe this might be a permissions issue, please double-check the
> npm ERR! permissions of the file and its containing directories, or try running
> npm ERR! the command again as root/Administrator.
> 
> npm ERR! A complete log of this run can be found in:
> npm ERR!     /home/USER/.npm/_logs/2021-06-08T19_07_29_216Z-debug.log

Formerly with the same result, `npm` suggested to upgrade from 7.12.1 → 7.16.0 by running `npm install -g npm@7.16.0`, alas, that didn't change the issue with `wekan-5.30/bundle/programs/server/package.json`. There are suggestions to `chown` some dir's in order to fix `EACCES`, yet they remain questionable, as there is no obvious reason for that _new_ requirement.

**Comment by BabyFnord** [comment]
Bump @kimdiallo …

**Comment by BabyFnord** [comment]
Coming back to
> npm ERR! Error: EACCES: permission denied, open '/home/USER/wekan/wekan-5.30/bundle/programs/server/package.json'

To fix _this_, run:
```
chmod -R ug=rwX ~/wekan
chmod -R o= ~/wekan
```

Subsequently, update_wekan had another error in the line with supervisord failing to restart the wekan service. Checking `cat ~/logs/supervisord.log` showed
> ## There is an issue with `node-fibers` ##
> `/home/yourUberspaceName/wekan/wekan-5.80/bundle/programs/server/node_modules/fibers/bin/linux-x64-72-glibc/fibers.node` is missing.

Because `fibers.node` _is_ already there, [we need to rebuild it to make it work](https://stackoverflow.com/questions/13327088/meteor-bundle-fails-because-fibers-node-is-missing). Here's how, like after a consistently failing _update_wekan --rebuild_:
```
cd into the fresh wekan folder, ie. /wekan-5.80/bundle/programs/server/
rm node_modules/.bin/node-gyp
rm node_modules/.bin/node-pre-gyp
npm uninstall fibers
npm install fibers
npm audit fix --force
supervisorctl stop wekan
supervisorctl update wekan
supervisorctl start wekan
```
At this point, supervisorctl should report wekan to be up and running. While this got me from 5.28 to 5.80, I still have to find a way to implement this as a fix for v1.7 :)

**Comment by BabyFnord** [comment]
v1.8 of this script addresses above issues, tested working.
